### PR TITLE
Position manager moveLiquidity bug

### PR DESCRIPTION
**Pull request by grandizzy**
- fix order of params and pass price instead index to `Buckets.lpsToQuoteToken` in PositionManager.moveLiquidity
- change test: add collateral into bucket to exercise full exchange rate calculation

### CODE REVIEW

File: src/base/PositionManager.sol [hunk 1]
```diff
@@ -113,12 +114,12 @@ contract PositionManager is IPositionManager, Multicall, PositionNFT, PermitERC2
         IPool pool = IPool(params_.pool);
         (uint256 bucketLPs, uint256 bucketCollateral, , uint256 bucketDeposit, ) = pool.bucketInfo(params_.fromIndex);
         (uint256 maxQuote, , ) = Buckets.lpsToQuoteToken(
-            bucketLPs,
             bucketCollateral,
+            bucketLPs,
```
Comment (EdNoepel) [inline src/base/PositionManager.sol]: I'd fix this by changing the interface and implementation accordingly.  A method named `lpsToQuoteToken` should take LPB, not collateral, as the first argument.

Comment (grandizzy) [reply src/base/PositionManager.sol]: 👍good call, will do

Comment (grandizzy) [reply src/base/PositionManager.sol]: done with https://github.com/ajna-finance/contracts/pull/368/commits/034d75f7e7f6055ed1588354b863134a3e7585f6

**Review by EdNoepel — APPROVED** [review]
LGTM

**Pull request merged**